lib/net.js

```diff
@@ -146,7 +146,7 @@ function Socket(options) {
   } else if (options.fd !== undefined) {
     this._handle = createHandle(options.fd);
     this._handle.open(options.fd);
-    if ((options.fd == 1 || options.fd == 2) &&
+    if ((options.fd === 1 || options.fd === 2) &&
```

Comment:
This could conceivably be a breaking change for some. Since `options.fd` can be provided by the end user, it may be a string. Before this change, passing in `'1'` would have the same effect as `1`. But now it won't. It only affects whether stdout and stderr are blocking or not on Windows. ¯\\_(ツ)_/¯ @nodejs/platform-windows